### PR TITLE
Bunch of small fixes spotted whilst coming up to speed with the project

### PR DESCRIPTION
**Pull request by klochowicz**
My apologies for such a non-focussed PR, but this is all the small things I have spotted along the way when I was coming up to speed with the project.

I'd be happy to split it into separate PRs, but I'm not sure whether it's worth it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,6 @@ mobile/ios/Runner/libnative.a
 # ignore data runtime data
 data/**
 !data/coordinator/regtest/seed
+
+# ignore CLion/Android Studio files
+.idea/
```
Comment (luckysori) [inline .gitignore:33]: 🙃 Missing newline here.

**Review by luckysori — APPROVED** [review]
Thanks!

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+# Additional information for developers
+
+More detailed information about 10101.
+Aimed primarily for developers.
+
+## Flutter Rust Bridge
```
Comment (da-kami) [inline DEVELOPMENT.md:6]: 🙃 I think this should be a sub-section in `## 10101 app`

**Review by da-kami — COMMENTED** [review]
I like the idea of splitting the README in a general readme and `dev` instructions, but we should probably cleanup the general README more then. 

At the moment I feel the information is split up between `README` and `DEVELOPMENT` - maybe we can move all the dev stuff from `README` to `DEVELOPMENT`?

**Comment by klochowicz** [comment]
> I like the idea of splitting the README in a general readme and `dev` instructions, but we should probably cleanup the general README more then.

I had to draw the line somewhere, I feel this is a step forward nevertheless (less details that became semi-obsolete/unneeded with recent additions to `justfile`)

> 
> At the moment I feel the information is split up between `README` and `DEVELOPMENT` - maybe we can move all the dev stuff from `README` to `DEVELOPMENT`?

I'm happy for this, I think as long as the user can "Quickstart" to start the app without much fuss then it's enough. 
Perhaps we could do another PR with further cleanup?

**Comment by klochowicz** [comment]
merging this, as @da-kami is keen to write some follow-ups. 
@bonomat @holzeis if you have some feedback, I'll address it in the follow-up.

**Pull request merged**